src/components/App.jsx

```diff
@@ -11,7 +12,7 @@ import MyActivities from '../containers/MyActivities';
  * @return {jsx} React node for the entire application
  */
 const App = () => (
-  <div>
+  <React.Fragment>
```

Comment:
For consistency with other parts of the code base you could just `import React, { Fragment } from 'react';`